test/adapters/flexibleLeverageStrategyAdapter.spec.ts

```diff
@@ -838,10 +879,12 @@ describe("FlexibleLeverageStrategyAdapter", () => {
   describe("#rebalance", async () => {
```

Comment:
Can we write a test where theres 2 exchanges, we call rebalance on the 1st exchange, and then call rebalance on the second exchange right after, so the 2nd transaction would revert since rebalance interval has not elapsed

Reply:
Then also a passing case where we're above or below the bounds and timestamps are updated correctly